deploy/helm/templates/addons/migration-addon.yaml

```diff
@@ -18,7 +18,7 @@ spec:
     {{- if hasPrefix "oci://" .Values.addonChartLocationBase }}
     chartLocationURL: {{ .Values.addonChartLocationBase }}/dt-platform
     {{- else }}
-    chartLocationURL: {{ .Values.addonChartLocationBase }}/dt-platform-0.1.0.tgz
+    chartLocationURL: {{ .Values.addonChartLocationBase }}/dt-platform-0.1.1-test.1.tgz
```

Comment:
Use test chart?

Reply:
also need to update line 26 for chart version